src/libcore/macros/mod.rs

```diff
@@ -1044,7 +1044,7 @@ pub(crate) mod builtin {
         };
     }
 
-    /// Includes a utf8-encoded file as a string.
+    /// Includes a UTF-8-encoded file as a string.
```

Comment:
```suggestion
    /// Includes a UTF-8 encoded file as a string.
```

Reply:
Will fix, but I'm curious: can you point me towards a grammar reference that indicates that this is the right choice?

Reply:
It's not a grammar fix, just a readability fix. I never seen UTF-8 and encoded joined together in my life.

Reply:
So like, the `-` wasn't even strictly necessary in the first place. For one example of a resource talking about this, see https://apastyle.apa.org/learn/faqs/when-use-hyphen

Basically all of these rules are about clarity. Does the hypen disambiguate, or not? I would argue that it's quite clear without the hyphen.

Reply:
(I should note that I particularly over-use the hypen (oh my I just did it again in this sentence) so this is very much often my fault in Rust docs, though I didn't author this one...)